.travis.yml

```diff
@@ -18,3 +18,4 @@ script:
 - if [ "$TRAVIS_OS_NAME" = linux ]; then curl -sL https://github.com/phracker/MacOSX-SDKs/releases/download/10.13/MacOSX10.13.sdk.tar.xz | tar -Jxf -; export COREAUDIO_SDK_PATH="$PWD/MacOSX10.13.sdk"; fi
 - RUSTFMT=rustfmt cargo build --verbose --target=x86_64-apple-darwin
 - if [ "$TRAVIS_OS_NAME" = osx ]; then RUSTFMT=rustfmt cargo test --verbose; fi
+- if [ "$TRAVIS_OS_NAME" = osx ]; then rustup target add aarch64-apple-ios armv7-apple-ios armv7s-apple-ios; for i in aarch64-apple-ios armv7-apple-ios armv7s-apple-ios; do RUSTFMT=rustfmt cargo build --verbose --target=$i; done; fi
```

Comment:
This should probably move to [a matrix config](https://docs.travis-ci.com/user/customizing-the-build/#build-matrix), but that seems like a task for another time.